src/bin/commands/setup/__tests__/inquirer.js

```diff
@@ -1,29 +1,32 @@
-import test from "ava"
+import { join } from "path"
 import suppose from "suppose"
 import fs from "fs"
 
 // This test created to test inquirer public API
-test.cb("test inquirer interface", (t) => {
-  suppose(
-    "node",
-    [ "./stub/spawn-prompt.js" ],
-    { debug: fs.createWriteStream("debug.txt") }
-  )
-  .when(/In 1 word describe phenomic/, "Awesome\n")
-  .on("error", function(error) {
-    console.error(error)
-    t.fail()
-    t.end()
-  })
-  .end(function(code) {
-    if (code !== 0) {
-      console.error("Script exited with code: " + code)
-      t.fail()
-      t.end()
-    }
-    else {
-      t.pass()
-      t.end()
-    }
+describe("bin > commands > setup > inquirer", () => {
```

Comment:
They are not mandatory. They just there to help us know where are those test result comes from